server/routerlicious/packages/services/src/redis.ts

```diff
@@ -81,10 +96,27 @@ export class RedisCache implements ICache {
 		}
 	}
 
+	/**
+	 * Get a list of keys that have a given prefix.
+	 *
+	 * @param keyPrefix - Prefix for the keys to get.
+	 */
+	public async keysByPrefix(keyPrefix: string): Promise<string[]> {
+		return this.client.keys(`${this.getKey(keyPrefix)}*`);
```

Comment:
>`${this.getKey(keyPrefix)}*` [](http://example.com/codeflow?start=26&length=28)

Should this be `${keyPrefix}:*`?

Reply:
I think this is a different prefix :) the prefix else where were to avoid name collision..this prefix is to make sure that we get all the keys starting with a particular string

For instance for ephemeral containers if we have keys like A/B/C/1.blob, A/B/C/2.blob and we want to delete everything in A/B/C, then we can use this to fetch all the keys that need to be deleted

Reply:
Right, these two prefixes are different

Reply:
I see...in this case, shouldn't we need to allow the caller to specify `prefixOverride`?